gammapy/spectrum/tests/test_make.py

```diff
@@ -49,6 +49,11 @@ def spectrum_dataset_maker_crab():
     return SpectrumDatasetMaker(region=region, e_reco=e_reco, e_true=e_true)
 
 
+@pytest.fixture()
+def safe_mask_maker():
+    return SafeMaskMaker()
```

Comment:
Remove fixture, just put `SafeMaskMaker()` below.
Fixtures are only useful for resources that take time to produce (load from disk, compute) or lots of code.
Otherwise they are bad, because one has to scroll up and down from the tests to see what it is.

Reply:
Done